runtime/js/90_deno_ns.js

```diff
@@ -60,8 +65,35 @@ const denoNs = {
   makeTempDir: fs.makeTempDir,
   makeTempFileSync: fs.makeTempFileSync,
   makeTempFile: fs.makeTempFile,
-  cpuUsage: () => {
+  cpuUsage: (previousValue) => {
     const { 0: system, 1: user } = op_runtime_cpu_usage();
+    if (previousValue) {
+      if (typeof previousValue.user !== "number") {
+        throw new TypeErrorConstructor(
+          'The "previousValue.user" property must be of type number.',
+        );
+      }
+      if (typeof previousValue.system !== "number") {
+        throw new TypeErrorConstructor(
+          'The "previousValue.system" property must be of type number.',
+        );
+      }
+      if (previousValue.user < 0 || !NumberIsFinite(previousValue.user)) {
+        throw new RangeError(
```

Comment:
@kt3k I looked at using `RangeErrorConstructor` here but that looks to be identical to `TypeErrorConstructor,` which is why I've gone with `new RangeError` instead 👍

Reply:
Hey @JakeChampion Thanks for the PR. Can we move these implementations to `ext/node/polyfills/process.ts` instead of implementing here.

I guess we are more interested in making `process.cpuUsage()` compatible with Node, and less interested in extending `Deno.cpuUsage`. If we only change `process.cpuUsage()`, we can make progress more quickly. (Also we can use node compatible validators (`validateObject`, `validateNumber`) and error classes (`ERR_INVALID_ARG_VALUE.RangeError`) if we implement there).

ref: https://github.com/nodejs/node/blob/5812a61a68d50c65127beb68dd4dfb0242e3c5c9/lib/internal/process/per_thread.js#L129-L143